packages/homepage/content/changelog/1-5-0/index.md

```diff
@@ -0,0 +1,24 @@
+---
+banner:
+title: v1.5.0
+slug: 1-5-0
+date: 2017-08-16
+description: A new editor and offline support
+---
+
+###
```

Comment:
What is this header on the top for? Not sure if we can remove this 🤔

Reply:
consistent spacing with those that do have titles